jobs/ci_open_jenkins/build/customs.groovy

```diff
@@ -20,9 +20,13 @@ private Job microservice(String serviceName) {
 
 microservice(customsInventoryLinkingImports)
 microservice(customsInventoryLinkingExports)
-microservice(customsDeclarations)
 microservice(customsNotification)
 microservice(customsApiCommon)
+new SbtMicroserviceJobBuilder(customsDeclarations).
+        withSCoverage().
+        withScalaStyle().
```

Comment:
probably a very brief comment here to say it is a fix to enable zip task in DECs to work